app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -35,7 +35,7 @@ def show
     param :product_id, :number, :required => true, :desc => N_("ID of a product to list repository sets from")
     def available_repositories
       scan_cdn = sync_task(::Actions::Katello::RepositorySet::ScanCdn, @product, @product_content.content.id)
-      repos = scan_cdn.output[:results]
+      repos = scan_cdn.output[:results] || []
```

Comment:
@jlsherrill the server logs give this:

```
[foreman-tasks/action] [E] CDN loading error: access forbidden to https://cdn.redhat.com:443/content/dist/rhel/server/7/7Server/listing (Katello::Errors::SecurityViolation)
```

But the `output[:results]` is still set to nil instead of an empty array. This causes hammer to report a call of `select on NilClass` due to [this code](https://github.com/Katello/katello/blob/master/app/controllers/katello/api/v2/repository_sets_controller.rb#L40) in Katello.